binr/r2r/r2r.c

```diff
@@ -966,7 +972,7 @@ static void interact_commands(R2RTestResultInfo *result, RPVector *fixup_results
 		}
 	}
 
-	replace_cmd_kv (result->test->path, test->cmds.line_begin, test->cmds.line_end, "CMDS", newcmds, fixup_results);
+	replace_cmd_kv_file (result->test->path, test->cmds.line_begin, test->cmds.line_end, "CMDS", newcmds, fixup_results);
```

Comment:
what about kvfile() instead of kv_file()

Reply:
File has nothing to do with the kv directly. Theoretically `replacecmdkv_file` would make more sense than `replace_cmd_kvfile`.

Reply:
it is a key-value file. anyway just a matter of taste